rules/python_rules.build_defs

```diff
@@ -447,7 +476,8 @@ def pip_library(name:str, version:str, hashes:list=None, package_name:str=None,
 def python_wheel(name:str, version:str, hashes:list=None, package_name:str=None, outs:list=None,
                  post_install_commands:list=None, patch:str|list=None, licences:list=None,
                  test_only:bool&testonly=False, repo:str=None, zip_safe:bool=True, visibility:list=None,
-                 deps:list=[], name_scheme:str=None, strip:list=['*.pyc', 'tests']):
+                 deps:list=[], name_scheme:str=None, strip:list=['*.pyc', 'tests'],
+                 use_package_root:bool=CONFIG.PYTHON_USE_PACKAGE_ROOT):
```

Comment:
I'm not sure why we need this here? Third party code can already be imported via the root path.

Reply:
I responded where it's used.